docs/cbmc.md

```diff
@@ -0,0 +1,15 @@
+# CBMC Proof Setup
+
+1. In your GNU compatible environment, open a terminal.
+
+1. Ensure that the Git submodule under \<SRC_ROOT\>/tools/aws-templates-for-cbmc-proofs has been cloned.
+
+1. Change the current working directory to the library's cbmc folder:  
+   `cd <SRC_ROOT>/libraries/standard/<LIBRARY_NAME>/cbmc`
+
+1. Run the setup.py script in aws-templates-for-cbmc-proofs with Python 3.6+:  
+   `Python3 ../../../../aws-templates-for-cbmc-proofs/scripts/setup.py`  
+   When prompted for the source root set either the relative path or the absolute path:  
+   `What is the path to the source root: ../../../..`
+
+Setting up a new proof and other information about the aws-templates-for-cbmc-proofs can be found at https://github.com/awslabs/aws-templates-for-cbmc-proofs.
```

Comment:
How to download and run CBMC is to be verified first for the public README.md (we have internal instructions on our compant wiki) and added later.

Reply:
Nice!

Reply:
None of these instructions should be required now that the templates repository is submoduled and will be grabbed by the normal git submodule update, and now that the symbolic links are part of the current repository.  The only thing that might be necessary is how to install cbmc and cbmc-viewer, and how to run setup-proofs.py to start a new proof.

Reply:
Ok we shall delete these in anticipation of the sub-module going public.